libs/counterfactuals/src/lib/CounterfactualChart.tsx

```diff
@@ -814,7 +814,7 @@ export class CounterfactualChart extends React.PureComponent<
     isString: boolean,
     newValue?: string | number
   ): void => {
-    if (!this.temporaryPoint || !newValue) {
+    if (!this.temporaryPoint || (!newValue && newValue !== 0)) {
```

Comment:
Are you checking for `newValue === undefined`?

Reply:
Yes, and also `newValue ===“”` when the value is empty string